djangosaml2/conf.py

```diff
@@ -26,7 +26,8 @@
 from djangosaml2.utils import get_custom_setting, get_endpoints
 
 
-BASE_PATH = os.path.dirname(os.path.dirname(djangosaml2.__file__))
+BASE_PATH = settings.SAML2_IDP_BASE_DIR if settings.SAML2_IDP_BASE_DIR else os.path.dirname(
```

Comment:
This will cause a AttributeError if the `SAML2_IDP_BASE_DIR` is not present. We need to use getattr here.